en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc

```diff
@@ -280,6 +361,36 @@ a|
 
 |===
 
+==== Shopify
+[%header%autowidth.spread]
+
+|===
+
+|Step |Field |Example Value |Notes
+.9+|Connect to your HTTP Account |*Connection Display Name* | `+Shopify at dpelaez+shopifydev@mulesoft.com (QA)+` | For more information, see the https://partners.shopify.com/[Shopify Partner Program^].
```

Comment:
This should be
`Shopify at my.account@mail.com`

Reply:
May I suggest `my.account@example.com`? Makes it even more obvious that it's an example